### ISSUE TITLE
[Android] outputPath is null, need to remove isEmpty check

### ISSUE THREAD

**Issue by danieldunderfelt** [bamlab/react-native-image-resizer#19]
Hi!

In using `image-resizer`, the Android build crashed when running the `createResizedImage` function. Specifically, on line 112 of `ImageResizer.java`, the `isEmpty()` check fails if I don't set an `outputPath`. This conditional:

```
if (outputPath != null || !outputPath.isEmpty()) {
  path = new File(outputPath);
}
```
Needs to be:
```
if (outputPath != null) {
  path = new File(outputPath);
}
```

Android Studio actually highlights the second part of the if statement saying that "outputPath is always null". I don't use an outputPath as saving to cache suits me perfectly. But the implementation seems broken.

Thanks for a great and focused library!

**Comment by LzxHahaha** [comment]
+1

**Comment by stoffern** [comment]
@LzxHahaha it will error if you have .isEmpty()
this was fixed here: 3b8d1c1940a3dfdf7cffd9206f669f68d5fb3696

**Comment by 4ian** [comment]
Closing this issue as it's fixed :)
